FCC Survey Page/css/sp-css.css

```diff
@@ -0,0 +1,113 @@
+@import url('https://fonts.googleapis.com/css?family=Noto+Sans+JP|Roboto+Slab&display=swap');
+
+html,
+body {
+    background-color: #A9D7D1;
+}
+
+#form-outer {
+    border: 5px;
+    border-radius: 10px;
+    max-width: 65%;
+    margin: 0 auto;
+    background-color: #FAFAFA;
+    padding: 0.5em;
+}
+
+#title {
+    font-family: 'Noto Sans JP', sans-serif;
+    font-size: 2em;
+    text-align: center;
+ }
+
+#description {
+    margin: 1.2em 0 1.2em 0;
+    padding: 0.5em;
+    border: 4px;
+    border-radius: 17px;
+    background-color: #fcca5d;
+    font-size: 1.2em;
+    font-family: 'Roboto Slab', serif;
+    text-align: center;
+}
+
+input#name, input#email, input#age {
```

Comment:
All good here. 
From design point of view. I would add some **padding**, reduce the **border** to 1px and make the **border-radius** bigger as it follows:
```
border: 1px solid rgb(161, 135, 79);
border-radius: 5px;
padding: 5px 7px;
```

Suggestion: You can apply these styles to the rest of the fields like Age, Other and Comment.